msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -1524,6 +1528,131 @@ The following code provides an example of meeting end event payload:
+1. In the **Meeting event subscriptions** section, select the events:
+    * Participant join
+    * Participant leave
+1. Select **Save**
+
+   :::image type="content" source="~/assets/images/apps-in-meetings/participant-events.png" alt-text="Screenshot shows how developer portal display for participant events.":::
+1. Ensure that the `OnlineMeetingParticipant.Read.Chat` RSC permission is configured in your app manifest.
+
+   If your app doesn't have the RSC permission, add it through the **Configure** > **Permissions** section of your app in Developer Portal. For more information, see [RSC permissions.](~/graph-api/rsc/resource-specific-consent.md)
+
+Following are the examples of the participant join and leave event payloads:
+
+# [Participant join event](#tab/participant-join-event) 
+
+The following code provides an example of the participant join event payload:
+
+```json
+{ 
+
+    "type": "event", 
+    "name": "application/vnd.microsoft.meetingParticipantJoin",
+    "timestamp": "2023-02-23T19:34:07.478Z", 
+    "channelId": "msteams", 
+    "serviceUrl": "https://smba.trafficmanager.net/amer/", 
+    "from": { 
+        "id": "29:id_xyz" 
+    }, 
+    "conversation": { 
+        "isGroup": true, 
+        "conversationType": "groupchat", 
+        "id": "19:meeting_threadId@thread.v2" 
+    }, 
+"recipient": { 
+    "id": "28:botid" 
+    },  
+    "value": { 
+       "members": [ 
+       { 
+        "user": { 
+            "tenantId": "tenantid", 
+            "objectId": "user_object_Id", 
+            "id": "29:userId ", 
+            "name": "Test User", 
+            "aadObjectId": " user_object_Id " 
+        },   
+        "meeting": { 
+            "inMeeting": true, 
+            "role": "Organizer" //Atendee, Organizer, Presenter 
+        },  
+        }], 
+    }, 
+    "channelData": { 
+        "tenant": { 
+            "id": "tenantId" 
+        }, 
+        "meeting": { 
+            "id": "encoded_meetingId" 
+        } 
+    } 
+} 
+```
+
+# [Participant leave event](#tab/participant-leave-event) 
+
+The following code provides an example of the participant leave event payload:
+
+```json
+{ 
+
+    "type": "event", 
+    "name": "application/vnd.microsoft.meetingParticipantLeave ",
+    "timestamp": "2023-02-23T19:34:07.478Z", 
+    "channelId": "msteams", 
+    "serviceUrl": "https://smba.trafficmanager.net/amer/", 
+    "from": { 
+        "id": "29:id_xyz" 
+    }, 
+    "conversation": { 
+        "isGroup": true, 
+        "conversationType": "groupchat", 
+        "id": "19:meeting_threadId@thread.v2" 
+    }, 
+"recipient": { 
+    "id": "28:botid" 
+    },  
+    "value": { 
+       "members": [ 
+       { 
+        "user": { 
+            "tenantId": "tenantid", 
+            "objectId": "user_object_Id", 
+            "id": "29:userId ", 
+            "name": "Test User", 
+            "aadObjectId": " user_object_Id " 
+        },   
+        "meeting": { 
+            "inMeeting": true
```

Comment:
Can you confirm with Ryan that the inMeeting should be false as the participant already left the meeting?

Reply:
Yes, this needs to be false.

Reply:
Updated